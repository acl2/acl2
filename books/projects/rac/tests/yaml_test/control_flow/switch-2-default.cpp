// RAC begin

int foo(int a) {
  int b = 0;

  switch (a) {
  default:
    b = 1;
    break;
  default:
    b = 2;
    break;
  }

  return b;
}

// RAC end
