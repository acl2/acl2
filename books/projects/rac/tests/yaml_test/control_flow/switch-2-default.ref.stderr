switch-2-default.cpp:7-9 (2-10): Duplicate default case
  7 |   default:
    |   ^^^^^^^^^
  8 |     b = 1;
    | ^^^^^^^^^^^
  9 |     break;
    | ^^^^^^^^^^
note: previous default was declared here:
 10 |   default:
    |   ^^^^^^^^^
 11 |     b = 2;
    | ^^^^^^^^^^^
 12 |     break;
    | ^^^^^^^^^^
